### ISSUE TITLE
PHP Memory

### ISSUE THREAD

**Issue by hurrycaner** [backbee/backbee-php#763]
How much php memory is needed to run backbee? i've noticed that this info is missing in the requirements section... i wanna check if it's possible to run this on a low end enviroment...

**Comment by eric-chau** [comment]
Hello @hurrycaner, in most of cases 128 mb is enough to run BackBee CMS.

**Comment by eric-chau** [comment]
You're right, this info is missing in the requirements, we will add it soonly, thank you!
